deps.edn

```diff
@@ -126,6 +126,7 @@
   org.apache.poi/poi-ooxml                  {:mvn/version "5.2.5"
                                              :exclusions  [org.bouncycastle/bcpkix-jdk15on
                                                            org.bouncycastle/bcprov-jdk15on]}
+  org.apache.poi/poi-ooxml-full             {:mvn/version "5.2.5"}
```

Comment:
if there's a way not to include this dep, I'd be happier... it's a bit large

Reply:
What's the `-full` version being used for here?

If it's needed, I guess `poi-ooxml` isn't needed? From reading [this page](https://poi.apache.org/components/) it sounds like it's a strict subset.